src/modules/GroupChannelList/context/GroupChannelListProvider.tsx

```diff
@@ -17,7 +17,7 @@ import { PartialRequired } from '../../../utils/typeHelpers/partialRequired';
 
 type OnCreateChannelClickParams = { users: Array<string>; onClose: () => void; channelType: CHANNEL_TYPE };
 type ChannelListDataSource = ReturnType<typeof useGroupChannelList>;
-type ChannelListQueryParamsType = Omit<GroupChannelCollectionParams, 'filter'> & GroupChannelFilterParams;
+export type ChannelListQueryParamsType = Omit<GroupChannelCollectionParams, 'filter'> & GroupChannelFilterParams;
```

Comment:
Do we have a reason for this?

Reply:
Yes! It's useful for customers when they manage the query params them self using the `useState`